app/routes.tsx

```diff
@@ -106,6 +108,20 @@ const routes: Routes = [
     layout: DashboardLayout,
     path: routePaths.APP,
     routes: [
+      ...(process.env.NODE_ENV === 'development' // TODO: Remove this!
```

Comment:
Guess same question as before, this'll be moved to the 'Contacts' page in some fashion?